packages/core/tests/svg.test.mjs

```diff
@@ -42,7 +43,7 @@ describe('Svg', () => {
     expect(svg.freeId).to.equal(0)
     expect(svg.body).to.equal('')
     expect(svg.style).to.equal('')
-    expect(svg.defs).to.equal('')
+    expect(svg.defs instanceof Defs).to.equal(true)
```

Comment:
there's a handy assertion you can use here that's a little more readable. `expect(svg.defs).to.be.an.instanceof(Defs)`. It'll throw a better error